### DISCUSSION TITLE
Disable group swithing between monitors

### DISCUSSION THREAD

**Discussion by optimistic5** [qtile/qtile#4208] [Q&A] [answered]
I have two monitors setup.
I have one group opened on `monitor 1` and another group on `monitor 2`.
When I switch between groups they are jumping between monitors.
I would like to avoid it.
![image](https://user-images.githubusercontent.com/25054050/227287941-fd94da4e-c983-450a-8409-e79926508be6.png)

According to this [answer](https://groups.google.com/g/qtile-dev/c/BgR2_Qi2-PQ/m/a2jSyRlJBwAJ) and [documentation](http://docs.qtile.org/en/latest/manual/config/lazy.html#group-functions) I just need to add `toggle=False` to `toscreen`.
Which I did and it does not work. Or I miss something.

This is my part of config
```python
for i in groups:
    keys.extend([
        # mod1 + letter of group = switch to group
        Key([mod], i.name, lazy.group[i.name].toscreen(toggle=False),
            desc="Switch to group {}".format(i.name)),

        # mod1 + shift + letter of group = move focused window to group
        Key([mod, "shift"], i.name, lazy.window.togroup(i.name),
            desc="move focused window to group {}".format(i.name)),
    ])
```
**How to disable group switching between monitors?**
Thank you.

**Comment by elParaguayo** [comment, marked as answer]
`toggle=False` is not what you're looking for. When `toggle` is `True`, this will show the previous group if the selected group is already on the screen.

What you're trying to do is only have certain groups appear on specific screens. Have a look at this: http://docs.qtile.org/en/stable/manual/faq.html#how-can-i-get-my-groups-to-stick-to-screens

**Reply by optimistic5** [reply]
> When toggle is True, this will show the previous group if the selected group is already on the screen.
Sorry, did not get it. Could you please elaborate more.

I supposed that I'm wrong, but I hoped I would solve it with this little switch :smile: 
As I did not manage to run code from your link. I will try again then.
Thank you!
